Source/SvgDocument.cs

```diff
@@ -41,8 +44,30 @@ public override SvgOverflow Overflow
             get { return GetAttribute("overflow", false, SvgOverflow.Visible); }
         }
 
-        private static int GetSystemDpi()
+        /// <summary> Get System Dpi Function </summary>
+        public static Func<int> GetSystemDpi
         {
+            get => getSystemDpi ?? (getSystemDpi = GetSystemDpiDefault);

```

Comment:
I think this property should be kept simple.

```cs
get => getSystemDpi;
```

Reply:
No Problem, I can make it simple